AFNetworking/AFHTTPClient.m

```diff
@@ -36,6 +36,8 @@
 #endif
 
 #if defined(__IPHONE_OS_VERSION_MIN_REQUIRED)
+#import <WatchKit/WatchKit.h>
+#elif defined(__IPHONE_OS_VERSION_MIN_REQUIRED)
```

Comment:
this is exactly the same condition as the `if` condition - i think you meant to specify `__WATCH_OS` for the first one, right?